onboarding/geo/README.md

```diff
@@ -0,0 +1,334 @@
+
+- **[NASA AIRS](https://airs.jpl.nasa.gov/):** The Atmospheric Infrared Sounder (AIRS) aboard NASA's Aqua satellite measures temperature and humidity profiles in Earth's atmosphere, aiding in weather forecasting and climate research.
+
+### ESA Data Sets
+
+- **[ESA Sentinel-1](https://sentinel.esa.int/web/sentinel/missions/sentinel-1):** Sentinel-1 is a radar imaging mission that provides all-weather, day-and-night imagery. It is used for disaster monitoring, mapping changes in land cover, and analyzing deformations of the Earth's surface.
+
+- **[ESA Sentinel-2](https://sentinel.esa.int/web/sentinel/missions/sentinel-2):** Sentinel-2 offers high-resolution multispectral imagery, making it valuable for agriculture, forestry, and land cover classification. It provides data on vegetation health, crop monitoring, and land use planning.
+
+- **[ESA Copernicus Climate Change Service](https://climate.copernicus.eu/):** ESA's Copernicus Climate Change Service provides a suite of data sets for climate analysis. This includes information on temperature, sea level, and greenhouse gas concentrations, essential for climate monitoring and research.
+
+## Most Useful Data Sets for Climate and Agricultural Analysis
+
+For climate and agricultural analysis, several data sets stand out:
+
+- **[NASA MODIS](https://modis.gsfc.nasa.gov/):** MODIS data provides invaluable information for tracking changes in vegetation, land surface temperature, and cloud cover, making it a cornerstone of climate and agricultural research.
+
+- **[ESA Sentinel-2](https://sentinel.esa.int/web/sentinel/missions/sentinel-2):** Sentinel-2's high-resolution multispectral imagery is particularly useful for monitoring crop health, land use changes, and assessing the impact of climate on agricultural productivity.
+
+- **[ESA Copernicus Climate Change Service](https://climate.copernicus.eu/):** ESA's climate data sets, such as those related to temperature and greenhouse gases, are vital for climate scientists and agricultural researchers studying long-term trends and climate impacts on agriculture.
+
+Exploring these data sets and their applications is a fascinating journey into the world of satellite imagery and its diverse uses across various domains.
+
+#### Useful Links
+
+- [Datasets Article](https://gisgeography.com/free-satellite-imagery-data-list/)
+- [Satellite Imagery and Deep Learning](https://www.youtube.com/watch?v=CQlLa_UWncg)
+- [Deep Learning with Satellite Imagery Workshop](https://www.youtube.com/watch?v=3Xn21RT-y7Y)
+- [Introduction to Remote Sensing with Python](https://www.youtube.com/watch?v=gi4UdFsayoM)
+- [An ML Approach to Satellite Imagery](https://www.youtube.com/watch?v=5PNnPagENxQ)
+
+## Commonly Used Python GIS Libraries
```

Comment:
I would certainly add https://pyproj4.github.io/pyproj/stable/index.html this package is very very useful as discussed on rocket.chat.

another package few packages for plotting https://mapscaping.com/essential-geospatial-python-libraries/

What's your thought on not numbering but making a short list with bullets, keep it pretty short in terms of description. A bit similar to https://github.com/sshuair/awesome-gis . Why not number? Because numbering implies an order, I'm not sure that there is an order. Ofc, you could order by number of git stars, etc. If we do that, the question arises, what is the relevance and/or how do we keep it up to date.

Reply:
https://github.com/sshuair/awesome-gis
https://github.com/eric-erki/awesome-gis

 https://gdal.org/programs/index.html 
https://rasterio.readthedocs.io/en/latest/index.html